sdk/Cargo.toml

```diff
@@ -60,6 +60,7 @@ chrono = { version = "0.4.24", default-features = false, features = ["serde", "w
 ciborium = "0.2.0"
 conv = "0.3.3"
 coset = "0.3.1"
+cryptographic-message-syntax = "0.22.0"
```

Comment:
It looks like the ASN1 code has been more or less broken out from its original home in PyOxidizer into a standalone CMS crate that we can import here and it appears well-maintained (last push was ~2 weeks ago). Using this code allows us to update the bcder crate as well, which was blocking some other upgrades.